src/randomenv.cpp

```diff
@@ -46,6 +46,7 @@
 #if HAVE_DECL_GETIFADDRS
 #include <ifaddrs.h>
 #endif
+#ifndef __linux__
```

Comment:
Thanks!
Though, I'd slightly prefer to handle this in the build system, so that the symbol `HAVE_DECL_SYSCTL` doesn't get set in the first place in Linux. This avoids having to handle it on a case by case basis on all caller sites.

Reply:
Good point, fixed.